build.gradle

```diff
@@ -78,7 +78,7 @@ dependencies {
         compile 'com.epam.reportportal:commons-model:3.0.1'
```

Comment:
please, mention this issue the changelog
https://github.com/reportportal/reportportal/issues/201